packages/relay/src/lib/config/hbarSpendingPlanConfigService.ts

```diff
@@ -131,7 +131,9 @@ export class HbarSpendingPlanConfigService {
     const spendingPlanConfig = ConfigService.get('HBAR_SPENDING_PLANS_CONFIG') as string;
 
     if (!spendingPlanConfig) {
-      logger.trace('HBAR_SPENDING_PLANS_CONFIG is undefined');
+      if (this.logger.isLevelEnabled('trace')) {
```

Comment:
This line here causing the failure of the CIs. This class doesn't have a logger property. This could easily be fixed by using the logger instance of the arguments.
```suggestion
      if (logger.isLevelEnabled('trace')) {
```

Reply:
Nice catch @quiet-node.
@belloibrahv please run an `npm run build-and-test` before pushing to confirm everything builds and unit tests run well.
Looking forward to getting this PR in.
Thanks